<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Module 2: Digitalisation in Research</title>
    <meta charset="utf-8" />
    <meta name="author" content="Hannah Metzler   CSH Vienna &amp; Medical University of Vienna" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
    <link rel="stylesheet" href="libs/footer&middle.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Module 2: Digitalisation in Research
## Introduction to Transparency and Reproducibility in Research
### Hannah Metzler <br> <em>CSH Vienna &amp; Medical University of Vienna</em>
### 25.11.2021

---






layout: true

---

# Can humans predict the future? 
## History of the reproducibility crisis in psychology

* Bem 2011: Humans can predict in which location erotic images will appear

* False positive psychology: Analytic flexilibity 
  * Outlier exclusion
  * 
  * 
  
* False positive rate 6x%

---
layout: true
&lt;div class="my-footer"&gt;&lt;span&gt;
&lt;a&gt; Open Science Collaboration 2015; Klein et al. 2014; Camerer et al. 2018; Camerer et al. 2016; Hensel 2021; Gordon et al. 2020
&lt;/a&gt;&lt;/span&gt;&lt;/div&gt;

---

# Reproducibility in the social sciences

.pull-left[.center-left[
* [Psychology 2015](http://science.sciencemag.org/content/349/6251/aac4716): 100 random studies: 38% replicated

* 13 famous findings in [social psychology 2014](https://econtent.hogrefe.com/doi/10.1027/1864-9335/a000178): 38% replicated

* [Behavioural economics 2016](http://science.sciencemag.org/content/351/6280/1433): 61%

* [Social sciences 2018](https://www.nature.com/articles/s41562-018-0399-z): published in Nature &amp; Science 2010-2015: 50%-67%

* [SCORE project](https://royalsocietypublishing.org/doi/10.1098/rsos.200566): Expert pedictions
]]
.pull-right[.center-right[
Expected replication rates: 
&lt;img src="figures/Golden_scoreproject.svg" width="500" /&gt;
]]
    
    
???
[Management](https://www.sciencedirect.com/science/article/pii/S0263237321000025?dgcid=rss_sd_all)

---
layout: true
&lt;div class="my-footer"&gt;&lt;span&gt;
&lt;a&gt; Begley &amp; Ioannidis 2015; Errington et al. 2014; Prinz et al. 2011; Begley &amp; Ellis 2012;  Francis, 2014; Ioannidis et al. 2009;   Lancee et al. 2017;&lt;br&gt; Amara &amp; Neves 2021
&lt;/a&gt;&lt;/span&gt;&lt;/div&gt;

---

# Problems &amp; projects in biology &amp; medicine

* [Review for basic and preclinical biomedical research](http://circres.ahajournals.org/content/116/1/116)
* A few examples: 

  * [Cancer Biology](https://elifesciences.org/articles/04333), oncology, women's health, cardiovascular diseases: so far [5 of 17](https://www.nature.com/articles/d41586-021-02486-7?proof=t%C2%A0) highly cited articles, [25%](https://www.nature.com/articles/nrd3439-c1?linkId=33568131), [11%](https://www.nature.com/articles/483531a#Tab1)
  
  * [Micro-array based gene expression studies](https://pubmed.ncbi.nlm.nih.gov/19174838/): 10-40% replicate, &lt;br&gt; issues in [epigenetics](http://www.genetics.org/content/198/2/449), [DNA methylation methods](https://www.sciencedirect.com/science/article/pii/S2666389920300143)
  
  * Pharmacology of [antipsychotic drugs](https://www.nature.com/articles/tp2017203): 81% deviate from registered plan
  
  * [Brazilian Reproducibility Initiative](https://www.nature.com/articles/d41586-021-02486-7?proof=t%C2%A0) for biomedical research

---
layout: true
&lt;div class="my-footer"&gt;&lt;span&gt;
&lt;a&gt; Kapoor &amp; Narazanan 2021; Baker 2016
&lt;/a&gt;&lt;/span&gt;&lt;/div&gt;

---

.pull-left[
## 'Hard' sciences

* Computational reproducibility in [Machine Learning](https://reproducible.cs.princeton.edu/): 
  * 4 out of 12 papers 
  * [Checklist](https://www.cs.mcgill.ca/~jpineau/ReproducibilityChecklist.pdf) for reproducible ML

* [Survey](https://www.nature.com/articles/533452a) in natural sciences =&gt;
]

.pull-right[

&lt;img src="figures/Baker_survey_disciplines.png" width="380" /&gt;
]

---
layout:true
---

# Cognitive biases all humans have

* Selection bias: Seeking out information

* Confirmation bias: Evaluating information

* Bandwagon effect: agree to easily for group harmony

* Cluster illusion: Recognizing patterns where there are none

* Reporting bias: positive results

???
https://www.nature.com/articles/d42473-019-00004-y#ref-CR17

---

# Biases in Research

* HARKing: Hypothesizing after results are known


---

# Biases throughout the research process

&lt;img src="figures/cruwell_biases_cycle.png" width="380" /&gt;

* Design: specific task/stimuli/method… 
* Data collection: experimenter biases
  * Bright vs. dull rats (Rosenthal &amp; Lawson 1964)
  * Power priming
* Confirmation bias
* Data analysis: experimenter degrees of freedom 
  * Inclusion of data points, conditions, participants, analysis methods, …
  * Looking at results before stopping data collection
* Write up and publication bias: 
  * Selective submission and acceptance
  * Avoiding null-results &amp; non-replications

---



# Open Science Framework

&lt;img src="figures/research_lifecycle_original.png" width="500" /&gt;



---



# Pre-registration

https://osf.io/w82ms/wiki/home/
* A plan, not a prison

* OSF infos: 
https://www.cos.io/initiatives/prereg

Why? 
- prevents p-hacking
- clearly distinguishes exploratory from confirmatory research
- works against publication bias
Source: https://osf.io/yd487/ Schönbrodt, Scheel, Stachl, 2017

---

# Pre-registration in Pratice

Where? 

* [As predicted](www.aspredicted.org): Short template
* OSF: Detailed template
* Any public, open-access repository with time-stamped version control
  * Github, figshare...

As predicted slides 1 and 2

---

# Data management

---
layout: true
&lt;div class="my-footer"&gt;&lt;span&gt;
&lt;a href ="https://www.cos.io/rr"&gt;https://www.cos.io/rr/  &lt;/a&gt;
&lt;/span&gt;&lt;/div&gt;

---


# Registered Reports (RR)

.pull-left[.center-left[
* Peer-reviewed preregistration  &lt;br&gt; prior to data collection

&lt;!-- ```{r, echo=FALSE, out.width=800, fig.align='center'} --&gt;
&lt;!-- knitr::include_graphics("figures/reg_report.png") --&gt;
&lt;!-- ``` --&gt;

* publication independent of results

* protects against reviewer HARKing

* reduces bias &amp; increases rigor

_"Because the study is accepted in advance, the incentives for authors change from producing the most beautiful story to the most accurate one."_  Chris Chambers
]]

.pull-right[.center-right[
&lt;img src="figures/reg_reports_osf_graphic.png" width="470" style="display: block; margin: auto 0 auto auto;" /&gt;
]]


---
layout: true
&lt;div class = "my-footer"&gt; &lt;span&gt;
&lt;a href="https://doi.org/10.1016/j.cortex.2017.07.014"&gt; McIntosh (2017) Exploratory reports: A new article type for Cortex. 
&lt;/a&gt; &lt;/span&gt; &lt;/div&gt;

---

# Exploratory Reports

.pull-left[.center-left[
* Open research questions

* No strong predictions

* Generation of hypotheses

* Transparency for researchers, reviewers, readers
  * Open data &amp; materials
  * No story telling

]]

.pull-right[.center-right[

**Useful for e.g.** 
* data mining
* detailed single case descriptions
* clinical database for uncommon conditions
* true story of experiments

**Templates**
* [Neuroscience Journal Cortex](https://www.elsevier.com/__data/promis_misc/Exploratory_Reports_Guidelines.pdf)
* [Social Psychology](https://osf.io/mdz3g/) Journal IRSP
]]

---
layout: true
---

# Choosing journals for reproducible research

* Finding/evaluating journals:

    * Over 300 journals that accept [Registered Reports](https://www.cos.io/initiatives/registered-reports)
    
    * Journals that allow pre-registration: [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/)
    
    * [TOP factor](https://topfactor.org/): Ranked by transparency &amp; openess criteria ([about TOP](https://www.natureindex.com/news-blog/top-factor-rates-journals-on-transparency-openness))
    
    * [Scimagojr](www.scimagojr.com/journalrank.php): Rankings per discipline (not specifically open science)
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script src="libs/perc.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
